MFaaP.MFWSClient/MFWSVaultObjectSearchOperations.cs

```diff
@@ -82,9 +95,20 @@ public Task<ObjectVersion[]> SearchForObjectsByConditionsAsync(params ISearchCon
 		/// <param name="searchConditions">The conditions to search for.</param>
 		/// <returns>An array of items that match the search conditions.</returns>
 		public ObjectVersion[] SearchForObjectsByConditions(params ISearchCondition[] searchConditions)
+		{
+			return this.SearchForObjectsByConditions(0, searchConditions);
```

Comment:
Use a constant rather than hard-coding the default value.

Reply:
solution in net PR